README.md

```diff
@@ -62,140 +86,486 @@ public final class DroidDetails implements Query<Operation.Variables> {
+    }
+
+    public static final class Mapper implements ResponseFieldMapper<Data> {
+      final Factory factory;
+
+      final Field[] fields = {
+        Field.forList("feed", "feed", null, true, new Field.ObjectReader<Feed>() {
+          @Override public Feed read(final ResponseReader reader) throws IOException {
+            return new Feed.Mapper(factory.feedFactory()).map(reader);
+          }
+        })
+      };
+
+      public Mapper(@Nonnull Factory factory) {
+        this.factory = factory;
       }
 
-      public @Nullable String name() {
-        return this.name;
+      @Override
+      public Data map(ResponseReader reader) throws IOException {
+        final __ContentValues contentValues = new __ContentValues();
+        reader.read(new ResponseReader.ValueHandler() {
+          @Override
+          public void handle(final int fieldIndex, final Object value) throws IOException {
+            switch (fieldIndex) {
+              case 0: {
+                contentValues.feed = (List<? extends Feed>) value;
+                break;
+              }
+            }
+          }
+        }, fields);
+        return factory.creator().create(contentValues.feed);
       }
 
-      public @Nullable String classification() {
-        return this.classification;
+      static final class __ContentValues {
+        List<? extends Feed> feed;
       }
+    }
   }
 }
 ```
-
 ## Consuming Code
 
 You can use the generated classes with Retrofit to make requests to your GraphQL API.  Apollo includes a `ApolloConverterFactory` which when registered with Retrofit will offer reflection-free response to Java mapping.
 
 ```java
-interface ApiService {
-  @POST("/") Call<Response<DroidDetails.Data>> droidDetails(@Body OperationRequest<Operation.Variables> query);
+interface GithuntApiService {
+  @POST("graphql") Observable<Response<FeedQuery.Data>> githuntFeed(@Body FeedQuery query);
 }
 ```
 
-```java
-OkHttpClient okHttpClient = new OkHttpClient.Builder().build();
-Retrofit retrofit = new Retrofit.Builder()
-    .baseUrl(BASE_URL)
-    .client(okHttpClient)
-    .addConverterFactory(new ApolloConverterFactory.Builder().build())
-    .addConverterFactory(MoshiConverterFactory.create())
-    .build();
-ApiService service = retrofit.create(ApiService.class);
-service.droidDetails(new OperationRequest<>(new DroidDetails()))
-    application.service()
-            .droidDetails(new OperationRequest<>(new DroidDetails()))
-            .enqueue(new Callback<Response<DroidDetails.Data>>() {
-              @Override
-              public void onResponse(Call<Response<DroidDetails.Data>> call,
-                  retrofit2.Response<Response<DroidDetails.Data>> response) {
-                // use the graphql response which is now in Java Models.
-              }
+```
+    okHttpClient = new OkHttpClient.Builder()
+        .cache(new Cache(new File(getCacheDir(), "okhttp"), 10 * 1024))
+        .addNetworkInterceptor(loggingInterceptor)
```

Comment:
Definitely not needed, I don't know where this came from. I just removed it. Thanks for noticing it.